buttons/src/main/java/com/decathlon/vitamin/buttons/VitaminButtons.kt

```diff
@@ -4,38 +4,86 @@ import android.content.Context
 import android.util.AttributeSet
 import com.google.android.material.button.MaterialButton
 
-open class VitaminPrimaryButton @JvmOverloads constructor(
+open class VitaminMediumPrimaryButton @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
-    defStyleAttr: Int = R.attr.materialButtonStyle
+    defStyleAttr: Int = R.attr.buttonPrimaryMedium
 ) : MaterialButton(context, attrs, defStyleAttr)
 
-open class VitaminPrimaryReversedButton @JvmOverloads constructor(
```

Comment:
Personal opinion: `Vitamin{Type}{Size}Button` isn't better? I think I prefer to write first the type of the button before the size when I'm a consumer.

Reply:
Fixed